source/manuals/aws-tagging.html.md.erb

```diff
@@ -36,22 +36,20 @@ Currently, we do not enforce tags.
 
 In future, we may wish to consider mechanisms such as alerting on untagged resources, or automatically deleting untagged resources.
 
-## Tags used in GOV.UK Sign In
-
-GOV.UK Sign In is using the following tags:
-
 ### Mandatory
 
- - `Product`: should be `GOV.UK Sign In`
- - `System`: the name of the software system, for example `Authentication` or `Identity proofing and verification core`. Avoid abbreviations.
+ - `Directorate`: for example `GOV.UK` or `DSP`
```

Comment:
I'm unsure if 'Directorate' makes sense for tagging system stuff. Product felt like it'd make more sense. 🤔